### PR TITLE
Platinum Card Slide

### LINKED ISSUE #8: Add A Platinum Card Slide
### Slide name

PlatinumCard

### Slide information

This April we ran an April Fools prank where a bunch of HCB users received "platinum cards". For those who did (`data.individual.platinumCard` won't be null), it'd be nice to remind them of this. `data.individual.platinumCard` has the card's last four digits and it's associated organisation name. If you'd like to use styling from the main HCB application, let us know in `#hcb-internals`. 

Screenshot 2023-12-04 at 10 12 02 PM

### Code of Conduct

- [X] I agree to follow Hack Club's Code of Conduct

### PR DESCRIPTION
**Pull request by alx-alexpark**
Closes #8 

image

### CODE REVIEW

File: components/Wrapped/slides/PlatinumCardSlide.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100%"
+      }}
+    >
+      <h1
+        {...$.title({
+          // You can apply classes by daisy-chaining them after the $
+          // You can then add CSS properties like you
+          // normally would inside of this object
```
Comment (sampoder) [inline components/Wrapped/slides/PlatinumCardSlide.tsx]:
```suggestion
```

Small thing :D

File: components/Wrapped/slides/PlatinumCardSlide.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                fontSize: 16,
+                letterSpacing: 2
+              }}
+            >
+              PLATINUM
+            </p>
+          </div>
+          {/* <p className="absolute m0" style={{right: 0, marginRight: "1.5rem", marginLeft: "3.5rem"}}>
+      card name
+    </p> */}
```
Comment (sampoder) [inline components/Wrapped/slides/PlatinumCardSlide.tsx]:
```suggestion
```

Are we safe to delete this?

File: components/Wrapped/slides/PlatinumCardSlide.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        {...$.title({
+          // You can apply classes by daisy-chaining them after the $
+          // You can then add CSS properties like you
+          // normally would inside of this object
+          color: "white",
+          alignSelf: "start",
+          justifySelf: "flex-start"
+        })}
+      >
+        April Fools!
```
Comment (sampoder) [inline components/Wrapped/slides/PlatinumCardSlide.tsx]: Let's make this copy something a little more comprehensive. 

```suggestion
        On the first day of April, you  earned yourself a one-of-a-kind exclusive HCB Platinum Card. April Fools? 
```

Thoughts on that? @YodaLightsabr got any ideas?

File: components/Wrapped/slides/PlatinumCardSlide.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      }}
+    >
+      <h1
+        {...$.title({
+          // You can apply classes by daisy-chaining them after the $
+          // You can then add CSS properties like you
+          // normally would inside of this object
+          color: "white",
+          alignSelf: "start",
+          justifySelf: "flex-start"
```
Comment (sampoder) [inline components/Wrapped/slides/PlatinumCardSlide.tsx]: Let's have both the header & card centred with only a 8 to 16px gap between the two. Hope that makes sense? Can make a PR into this PR if it doesn't.

**Review by sampoder — CHANGES_REQUESTED** [review]
Wow! Great work on this and thanks for the quick turn around. I've dropped a couple of quick comments on the code. It might also be fun to add animations :D. We have some examples in the other slides using the helper.

File: components/Wrapped/slides/PlatinumCardSlide.tsx (hunk 3 above)
Comment (alx-alexpark) [reply components/Wrapped/slides/PlatinumCardSlide.tsx:25]: That sounds good

Comment (sampoder) [reply components/Wrapped/slides/PlatinumCardSlide.tsx]: cool!

**Review by sampoder — APPROVED** [review]
LGTM!

**Pull request merged**